### ISSUE TITLE
Feature request: Zettelkasten

### ISSUE THREAD

**Issue by devinat1** [different-ai/file-organizer-2000#21]
I really like the idea of this plugin. I use the Zettelkasten system, and the problem i am facing is with organizing MOCs (maps of content) which are themselves notes. These maps of content more or less take the job of the folder in that all notes with a specific subject link to that note instead of being buried inside a folder.

The use case I would like is the automatic organization of notes by linking them to their respective note. For example, if I have a MOC called "AI" and another one called "LLM", then LLM would link to AI, and everything linking to LLM would also link to AI etc.
So in other words, I would like file organization _through notes_ instead of through directories.

**Comment by devinat1** [comment]
To be more clear, I don't use tags and directories as the primary way to organize my vault. Rather, I have one directory called "Zettelkasten" and two tags (#idea and #MOC). The way my vault is organized is by linking notes to other notes. The issue I have is that it becomes very unclear which notes should be #idea notes and which should be #MOCs, and which MOCS should link to other MOCS and Zettel notes to MOCS etc. Utilizing AI to help with this would be very useful, and seems like an interesting extension to your plugin.

**Comment by devinat1** [comment]
I have one more note type actually which is #literature. It would be awesome if, for example, the moment I create a literature note with a given title, that note automatically links to one to many MOCS the title of the literature note is based on. The same thing with Zettels linking to MOCS as well.

**Comment by ashgansh** [comment]
sounds awesome

i'd love to have my "automatic" things going on but one of the main challenge is that it's extremely heavy on my openai credits.

are you self-hosting the product?

**Comment by devinat1** [comment]
Yes self hosting works for me. I agree that it would cost a lot of credits.
